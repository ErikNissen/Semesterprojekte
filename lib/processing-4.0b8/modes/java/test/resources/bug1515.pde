version https://git-lfs.github.com/spec/v1
oid sha256:622bf6fce52889188813d00026d0c816fdfce6a79cd0c6245781806d060c6007
size 223
