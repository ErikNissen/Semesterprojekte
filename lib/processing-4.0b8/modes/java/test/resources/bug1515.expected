version https://git-lfs.github.com/spec/v1
oid sha256:4df03183b1b2ab3a20687d148ec3f4a9ee6b063260548b01b87e2b0a51111707
size 867
